composer.json

```
@@ -46,6 +47,12 @@
         }
     },
     "extra": {
+        "patches": {
+          "drupal/embed": {
+            "Fix PSR-4 namespace for EmbedCKEditor5PluginBase (issue #3517882)":
+            "https://www.drupal.org/files/issues/2025-06-30/embed--2025-06-30--3517882--mr-26.patch"
```

Try keeping the same pattern for the patches we have in other components and ewcms, with the issue link:
```suggestion
            "https://www.drupal.org/project/embed/issues/3517882": "https://www.drupal.org/files/issues/2025-06-30/embed--2025-06-30--3517882--mr-26.patch"
```